bindings/nodejs/examples/how_tos/accounts_and_addresses/create_account.ts

```diff
@@ -18,26 +20,35 @@ async function run() {
             '.env STRONGHOLD_PASSWORD is undefined, see .env.example',
         );
     }
+    if (!process.env.STRONGHOLD_SNAPSHOT_PATH) {
+        throw new Error(
+            '.env STRONGHOLD_SNAPSHOT_PATH is undefined, see .env.example',
+        );
+    }
     if (!process.env.NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1) {
         throw new Error(
             '.env NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1 is undefined, see .env.example',
         );
     }
+    if (!process.env.WALLET_DB_PATH) {
+        throw new Error('.env WALLET_DB_PATH is undefined, see .env.example');
+    }
     try {
-        const walletOptions = {
-            storagePath: './alice-database',
+        const walletOptions: WalletOptions = {
+            storagePath: process.env.WALLET_DB_PATH,
             clientOptions: {
                 nodes: [process.env.NODE_URL],
             },
             coinType: CoinType.Shimmer,
             secretManager: {
                 stronghold: {
-                    snapshotPath: `./wallet.stronghold`,
-                    password: `${process.env.STRONGHOLD_PASSWORD}`,
+                    snapshotPath: process.env.STRONGHOLD_SNAPSHOT_PATH,
+                    password: process.env.STRONGHOLD_PASSWORD,
                 },
             },
         };
 
+        // Create the wallet
```

Comment:
Useless comment
```suggestion
```

Reply:
Sorta mimicks this: want me to copy the whole sentence or remove
https://github.com/iotaledger/iota-sdk/blob/develop/sdk/examples/how_tos/accounts_and_addresses/create_account.rs#L33

Reply:
I would say just remove, it's pretty obvious what options we provide